src/format/annotations/formatFailedTestsAnnotations.ts

```diff
@@ -1,39 +1,37 @@
 import { context } from '@actions/github';
 
 import { CreateCheckOptions } from './CreateCheckOptions';
-import { getFailedAnnotationsSummary } from './getFailedAnnotationsSummary';
 import { getFailedTestsAnnotationsBody } from './getFailedTestsAnnotationsBody';
 import { Annotation } from '../../annotations/Annotation';
-import { JsonReport } from '../../typings/JsonReport';
+import { TestRunReport } from '../../typings/Report';
 import { insertArgs } from '../../utils/insertArgs';
 import {
     failedTestsCheckName,
     testsFail,
-    testsSuccess,
     tooMuchAnnotations,
 } from '../strings.json';
 
 export const formatFailedTestsAnnotations = (
-    jsonReport: JsonReport,
+    runReport: TestRunReport,
     annotations: Array<Annotation>
 ): CreateCheckOptions => ({
     ...context.repo,
     status: 'completed',
     head_sha: context.payload.pull_request?.head.sha ?? context.sha,
-    conclusion: 'success',
+    conclusion: 'failure',
```

Comment:
@ArtiomTr This assumption is correct right?
If the tests failed, then the check should be marked failure.

Reply:
Yes, I think that this is right behavior. Thank you for your contribution!